docs/plugins/harbor.md

```diff
@@ -0,0 +1,93 @@
+# harbor Plugin
+
+This plugin installs [harbor](https://goharbor.io/) in an existing Kubernetes cluster using the Helm chart.
+
+## Usage
+
+##### External storage
+
+- Postgresql: Set the `database.type` to `external` and fill the information in `database.external` section.
+- Redis: Set the `redis.type` to `external` and fill the information in `redis.external` section.
+
+##### Disk storage
+
+Please be sure to change the `storageClass` in the options of the config to an existing StorageClass. You can refer to this [document](https://github.com/goharbor/harbor-helm#configure-how-to-persist-data)
+
+#### Network config
+
+This plugin support `Ingress`, `ClusterIP`, `NodePort`, `LoadBalancer` ，You can give choice to your needs.
+
+##### TLS config
+- Use self-signed certificates
+  1. Set `tls.enabled` to `true` and edit the corresponding domain name `externalURL`
+  2. Copy the self-signed certificate stored in the Pod `harbor-core` directory `/etc/core/ca` to your own PC
+  3. Trust the certificate on your own host
+- Using public Certificates
+  1. Add the public certificate as Secret
+  2. Set `tls.enabled` to `true` and edit the corresponding domain name `externalURL`
+  3. Configure `tls.secretName` to use the public certificate
+
+
+### Test/Local Dev Environment
+
+If you want **test plugin locally**，you can just use default params
+
+- Postgresql and Redis dependencies are automatically created
```

Comment:
I think this information should be put in the external storage section, so that if you are testing, you don't have to create these two dependencies. Otherwise, users might waste time creating these dependencies before reading on, then later only to find out these aren't mandatory for a testing env.